pytorch_lightning/loggers/tensorboard.py

```diff
@@ -265,9 +265,10 @@ def save(self) -> None:
 
     @rank_zero_only
     def finalize(self, status: str) -> None:
-        self.experiment.flush()
-        self.experiment.close()
-        self.save()
+        if status != "failed":
```

Comment:
I think we can still flush and close even if status is failed. Or is this not correct?

Reply:
`tests/utilities/test_cli.py::test_cli_distributed_save_config_callback` fails without the condition on commit dbbfef817a3fff52a4d0b98ef68ae39f03863f56.
I don't know why but somehow the problem was fixed on the current implementation.

Reply:
Is this test still a problem? I am its author